### ISSUE TITLE
Element Displayedness

### ISSUE THREAD

**Issue by Danp2** [w3c/webdriver#1285]
In chapter [C. Element Displayedness](https://w3c.github.io/webdriver/#element-retrieval#c-element-displayedness):
> This function is typically exposed to GET requests with a URI Template of /session/{session id}/element/{element id}/displayed. 

Suggest that this information also be listed under [13. Element State](https://w3c.github.io/webdriver/#element-state)

**Comment by andreastt** [comment]
It’s in the appendix because it relies on features that are not part of the web platform. I think it would be confusing to mix references to Selenium atoms in the definitions related to element state.

**Comment by foolip** [comment]
Some history on this issue I found:
https://github.com/w3c/webdriver/issues/660 (via https://github.com/w3c/webdriver/commit/4a1c3500b4872a2ecdb2a281446924c064e82ec7)
https://www.w3.org/2015/10/25-webdriver-minutes.html#action09 (via https://github.com/w3c/webdriver/commit/de7ca16463e558487562f699f8dff380ce5cb234)

Does any WebDriver implementation currently support the /displayed endpoint? https://w3c.github.io/webdriver/#element-displayedness isn't marked as non-normative, but there aren't any tests for it.

**Comment by Danp2** [comment]
> Does any WebDriver implementation currently support the /displayed endpoint? 

Yes, both Chromedriver and Geckodriver support this implementation. Haven't tested any others.
